ee_plugin/processing/add_image_collection.py

```diff
@@ -219,6 +221,14 @@ def buildDialog(self) -> QWidget:
         viz_layout.addWidget(self.viz_widget)
         viz_group.setLayout(viz_layout)
 
+        # --- Clip to Extent Checkbox ---
+        self.clip_checkbox = QCheckBox(_("Clip to Extent"))
+        self.clip_checkbox.setChecked(True)
+        self.clip_checkbox.setToolTip(
+            _("Whether to clip the final image to the specified extent.")
+        )
```

Comment:
```suggestion
        self.clip_checkbox = QCheckBox(
            _("Clip to Extent"),
            checked=True,
            toolTip=_("Whether to clip the final image to the specified extent.")
        )
```

nit: I think you could condense this